static/templates/stream_types.handlebars

```diff
@@ -17,11 +17,13 @@
             {{t '<b>Private, protected history:</b> must be invited by a member; new members can only see messages sent after they join; hidden from non-administrator users' }}
         </label>
     </li>
+    {{#if is_admin}}
     <li>
         <label class="checkbox">
             <input type="checkbox" name="is-announcement-only" value="is-announcement-only" {{#if is_announcement_only}}checked{{/if}}/>
             <span></span>
             {{t 'Restrict posting to organization administrators' }}
         </label>
     </li>
+    {{/if}}
```

Comment:
Have you tested this manually  I'd worry that this change would result in an error due to the browser not sending this value.

Reply:
I did not understand your doubt. Are you worried that browser won't send `is_announcement_only=True` when this is checked?
Anyway, I did check it manually and everything works fine. :)

Reply:
No, I'm worried it won't send anything in the case this field isn't part of the template.

Reply:
```python
1137     is_announcement_only = models.BooleanField(default=False)  # type: bool
```
The default value is false, so if nothing is sent, it does not cause a problem.